sys-apps/irqbalance/files/irqbalance.service

```diff
@@ -2,10 +2,8 @@
 Description=CPU Interrupt Balancer
 
 [Service]
-ExecStart=/usr/sbin/irqbalance --debug
-# --debug implies a lot of cruft
-# but it's the only way to get foregrounding
-StandardOutput=null
+EnvironmentFile=-/etc/irqbalance.env
```

Comment:
I don't understand.  Those guidelines say that it is discouraged, but this is an upstream supplied file, and not something I wrote for the ebuild (i.e., not downstream).  And anyway, the `irqbalance.env` file also sets environment variables that are not part of the command line ([see](https://github.com/Irqbalance/irqbalance/blob/master/misc/irqbalance.env) for yourself), which is precisely what the `EnvironmentFile` directive is for.

Or is the preferred way really to tell people to use their own `irqbalance.service.d/*.conf` files instead of the upstream file?  Or something else entirely?

BTW, after these changes, the only difference to the upstream supplied service unit is that the latter uses the deprecated `After=syslog.target` (at least on current master).  Would an alternative be to stop using the Gentoo specific service file and start using the upstream one instead (though it would require `sed`ing the path to `irqbalance.env`)?